backend/package-lock.json

```diff
@@ -1,8600 +1,8 @@
 {
   "name": "backend",
   "version": "1.0.0",
```

Comment:
Please don't commit `package-lock.json` (both for front and back), since dependencies were not changed.

Reply:
Ok, Thanks

Reply:
Could you please make sure that `package-lock.json` is not modified? If you have trouble doing that I could do that myself tomorrow. Thanks.

Reply:
Thanks, @Aivean So I noticed it was when I was doing the npm install setup it change the lockfileversion from 2 to 1. So I just reverted only the package-lock.json file for both frontend and backend back to the initial state.